test/types/fields-contentRelationship.types.ts

```diff
@@ -75,6 +78,25 @@ expectType<prismic.ContentRelationshipField<string, string, never, "filled">>(
 	},
 )
 
+/**
+ * Empty state with text.
+ */
+expectType<prismic.ContentRelationshipField>({
+	link_type: prismic.LinkType.Document,
+	text: "string",
+})
+expectType<prismic.ContentRelationshipField<string, string, never, "empty">>({
+	link_type: prismic.LinkType.Document,
+	text: "string",
+})
+expectType<prismic.ContentRelationshipField<string, string, never, "filled">>(
+	// @ts-expect-error - Filled fields cannot contain an empty value.
+	{
+		link_type: prismic.LinkType.Document,
+		text: "string",
+	},
+)
+
```

Comment:
❓ #ask: Should these tests be removed per https://github.com/prismicio/prismic-client/pull/352#discussion_r1758418970?